configs/config_mech.json

```diff
@@ -8,7 +8,7 @@
     "configurations": {
         "gnosis": {
             "agent_id": 37,
```

Comment:
Where is this used in the quickstart ? For the new mm, this doesn't really matters as far as I know.

Reply:
So this is just a fallback number. Ideally, the agent_id is picked from the staking program that you choose to stake in. But, if you run without staking then this agent_id is picked to mint the service.